tests/components/google/conftest.py

```diff
@@ -58,6 +58,35 @@
     "defaultReminders": [],
 }
 
+TEST_EVENT = {
```

Comment:
Can we make this a fixture?

Reply:
Would be nice, but options :)

Maybe for a future PR

Reply:
Post merge, following up on the fixture comment since i'm happy to explore it in a future PR.

This is used with `mock_events_list_items` which is a fixture. That is used at different points in the test to change the next response when events are listed, so the API is now something like:


```
    mock_events_list_items([TEST_EVENT])
```

or if testing a special case, something like this:

```
    mock_events_list_items(
        [
            {
                **TEST_EVENT,
                "id": "event-id-1",
                "iCalUID": "event-id-1@google.com",
                "summary": "All Day Event",
                "start": {"date": "2022-10-08"},
                "end": {"date": "2022-10-09"},
            },
        ]
    )
```

or for tests that are not using freezer:

```
    one_hour_from_now = dt_util.now() + datetime.timedelta(minutes=30)
    end_event = one_hour_from_now + datetime.timedelta(minutes=60)
    event = {
        **TEST_EVENT,
        "start": {"dateTime": one_hour_from_now.isoformat()},
        "end": {"dateTime": end_event.isoformat()},
    }
    mock_events_list_items([event])
```

So its definitely used as an input to a fixture, but its just a variable to help with creating new events for composing them.

So at best we could make another fixture that is a simpler fixture that already takes in the TEST_EVENT as a default and the caller passes in the deltas for the test, but I didn't think having multiple fixtures was worthwhile, and so that is why its composition via `TEST_EVENT`

If you have other ideas happy to explore them, but also this isn't a pain right now.

Thanks!